Description: Enable compile-time support for tcp-wrappers.
Author: Roger Light <roger@atchoo.org>
Index: mosquitto-0.11.3/config.h
===================================================================
--- mosquitto-0.11.3.orig/config.h	2011-07-08 14:58:15.000000000 +0100
+++ mosquitto-0.11.3/config.h	2011-07-08 14:58:54.000000000 +0100
@@ -10,7 +10,7 @@
 
 
 /* Uncomment to compile with tcpd/libwrap support. */
-//#define WITH_WRAP
+#define WITH_WRAP
 
 /* Compile with database upgrading support? If disabled, mosquitto won't
  * automatically upgrade old database versions. */
Index: mosquitto-0.11.3/config.mk
===================================================================
--- mosquitto-0.11.3.orig/config.mk	2011-07-08 14:58:54.000000000 +0100
+++ mosquitto-0.11.3/config.mk	2011-07-08 14:58:54.000000000 +0100
@@ -5,7 +5,7 @@
 #MANCOUNTRIES=en_GB
 
 CFLAGS=-I. -I.. -ggdb -Wall -O2 -I../lib
-LDFLAGS=
+LDFLAGS=-lwrap
 # Add -lwrap to LDFLAGS if compiling with tcp wrappers support.
 
 CC=gcc
