Description: Enable compile-time support for tcp-wrappers.
Author: Roger Light <roger@atchoo.org>
Index: mosquitto-0.10.1/config.h
===================================================================
--- mosquitto-0.10.1.orig/config.h	2011-05-30 19:46:42.000000000 +0100
+++ mosquitto-0.10.1/config.h	2011-05-30 19:46:46.000000000 +0100
@@ -10,7 +10,7 @@
 
 
 /* Uncomment to compile with tcpd/libwrap support. */
-//#define WITH_WRAP
+#define WITH_WRAP
 
 /* Compile with database upgrading support? If disabled, mosquitto won't
  * automatically upgrade old database versions. */
Index: mosquitto-0.10/config.mk
===================================================================
--- mosquitto-0.10.orig/config.mk	2011-04-29 22:07:29.950727002 +0100
+++ mosquitto-0.10/config.mk	2011-04-29 22:07:36.394727002 +0100
@@ -5,7 +5,7 @@
 #MANCOUNTRIES=en_GB
 
 CFLAGS=-I. -I.. -ggdb -Wall -O2 -I../lib
-LDFLAGS=
+LDFLAGS=-lwrap
 # Add -lwrap to LDFLAGS if compiling with tcp wrappers support.
 
 CC=gcc
