Description: Enable compile-time support for tcp-wrappers.
Author: Roger Light <roger@atchoo.org>
Forwarded: not-needed
--- a/config.h
+++ b/config.h
@@ -10,7 +10,7 @@
 
 
 /* Uncomment to compile with tcpd/libwrap support. */
-//#define WITH_WRAP
+#define WITH_WRAP
 
 /* Compile with database upgrading support? If disabled, mosquitto won't
  * automatically upgrade old database versions. */
--- a/config.mk
+++ b/config.mk
@@ -14,7 +14,7 @@
 	LIBS=
 endif
 
-LDFLAGS=
+LDFLAGS=-lwrap
 # Add -lwrap to LDFLAGS if compiling with tcp wrappers support.
 
 CC=gcc
