Description: Enable compile-time support for tcp-wrappers.
Author: Roger Light <roger@atchoo.org>
Index: mosquitto-0.12/config.h
===================================================================
--- mosquitto-0.12.orig/config.h	2011-07-25 22:13:19.772773001 +0100
+++ mosquitto-0.12/config.h	2011-07-25 22:13:22.488773001 +0100
@@ -10,7 +10,7 @@
 
 
 /* Uncomment to compile with tcpd/libwrap support. */
-//#define WITH_WRAP
+#define WITH_WRAP
 
 /* Compile with database upgrading support? If disabled, mosquitto won't
  * automatically upgrade old database versions. */
Index: mosquitto-0.12/config.mk
===================================================================
--- mosquitto-0.12.orig/config.mk	2011-07-25 22:13:26.932773002 +0100
+++ mosquitto-0.12/config.mk	2011-07-25 22:13:31.848773001 +0100
@@ -13,7 +13,7 @@
 	LIBS=
 endif
 
-LDFLAGS=
+LDFLAGS=-lwrap
 # Add -lwrap to LDFLAGS if compiling with tcp wrappers support.
 
 CC=gcc
