Description: Enable compile-time support for tcp-wrappers.
Author: Roger Light <roger@atchoo.org>
Index: mosquitto-0.9/config.h
===================================================================
--- mosquitto-0.9.orig/config.h	2010-11-27 00:27:10.000000000 +0000
+++ mosquitto-0.9/config.h	2010-11-27 00:27:15.000000000 +0000
@@ -6,7 +6,7 @@
  * ============================================================ */
 
 /* Uncomment to compile with tcpd/libwrap support. */
-//#define WITH_WRAP
+#define WITH_WRAP
 
 /* Compile with database upgrading support? If disabled, mosquitto won't
  * automatically upgrade old database versions. */
@@ -56,4 +56,4 @@
 #ifdef WIN32
 #define snprintf sprintf_s
 #define strcasecmp strcmpi
-#endif
\ No newline at end of file
+#endif
Index: mosquitto-0.9/config.mk
===================================================================
--- mosquitto-0.9.orig/config.mk	2010-11-27 00:27:13.000000000 +0000
+++ mosquitto-0.9/config.mk	2010-11-27 00:27:15.000000000 +0000
@@ -5,7 +5,7 @@
 #MANCOUNTRIES=en_GB
 
 CFLAGS=-I. -I.. -ggdb -Wall -O2 -I../lib
-LDFLAGS=-lsqlite3
+LDFLAGS=-lsqlite3 -lwrap -Wl,--as-needed
 # Add -lwrap to LDFLAGS if compiling with tcp wrappers support.
 
 CC=gcc
