Index: mosquitto-0.9/client/Makefile
===================================================================
--- mosquitto-0.9.orig/client/Makefile	2010-11-14 23:53:49.528854000 +0000
+++ mosquitto-0.9/client/Makefile	2010-11-14 23:53:53.992854002 +0000
@@ -2,7 +2,7 @@
 CFLAGS=-I../lib -Wall -ggdb
 LDFLAGS=-L../lib ../lib/libmosquitto.so.0
 INSTALL=install
-prefix=/usr/local
+prefix=/usr
 
 .PHONY: all install uninstall clean
 
Index: mosquitto-0.9/config.mk
===================================================================
--- mosquitto-0.9.orig/config.mk	2010-11-14 23:54:00.888854001 +0000
+++ mosquitto-0.9/config.mk	2010-11-14 23:54:06.788854000 +0000
@@ -15,6 +15,6 @@
 MSGFMT=msgfmt
 DOCBOOK2MAN=docbook2man.pl
 
-prefix=/usr/local
+prefix=/usr
 mandir=${prefix}/share/man
 localedir=${prefix}/share/locale
