Description: Install to /usr instead of /usr/local
Author: Roger Light <roger@atchoo.org>
Index: mosquitto-0.11.3/client/Makefile
===================================================================
--- mosquitto-0.11.3.orig/client/Makefile	2011-07-08 14:58:15.000000000 +0100
+++ mosquitto-0.11.3/client/Makefile	2011-07-08 14:58:54.000000000 +0100
@@ -2,7 +2,7 @@
 CFLAGS=-I../lib -Wall -ggdb
 LDFLAGS=-L../lib ../lib/libmosquitto.so.0
 INSTALL=install
-prefix=/usr/local
+prefix=/usr
 
 .PHONY: all install uninstall clean
 
Index: mosquitto-0.11.3/config.mk
===================================================================
--- mosquitto-0.11.3.orig/config.mk	2011-07-08 14:58:15.000000000 +0100
+++ mosquitto-0.11.3/config.mk	2011-07-08 14:58:54.000000000 +0100
@@ -15,6 +15,6 @@
 MSGFMT=msgfmt
 DOCBOOK2MAN=docbook2man.pl
 
-prefix=/usr/local
+prefix=/usr
 mandir=${prefix}/share/man
 localedir=${prefix}/share/locale
